pkg/host/internal/sriov/sriov.go

```diff
@@ -571,7 +574,7 @@ func (s *sriov) configSriovDevice(iface *sriovnetworkv1.Interface, skipVFConfigu
 	if err != nil {
 		return err
 	}
-	if pfLink.Attrs().OperState != netlink.OperUp {
+	if pfLink.Attrs().Flags&net.FlagUp == 0 {
```

Comment:
please put this one as part of the `netlinkLib` lets use it here and on the other place in the code